summingbird-storm/src/main/scala/com/twitter/summingbird/storm/BuildSummer.scala

```diff
@@ -46,19 +47,27 @@ object BuildSummer {
       case Some(cons) =>
         logger.debug("Node ({}): Using user supplied SummerConstructor: {}", dag.getNodeName(node), cons)
         cons.get
-      case None => legacyBuilder(storm, dag, node)
+      case None => legacyBuilder(storm, dag, node, jobID)
     }
   }
 
-  private[this] final def legacyBuilder(storm: Storm, dag: Dag[Storm], node: StormNode) = {
+  private[this] final def legacyBuilder(storm: Storm, dag: Dag[Storm], node: StormNode, jobID: JobId) = {
     val nodeName = dag.getNodeName(node)
     val cacheSize = storm.getOrElse(dag, node, DEFAULT_FM_CACHE)
     logger.info("[{}] cacheSize lowerbound: {}", nodeName, cacheSize.lowerBound)
 
+    val memoryCounter = counter(jobID, Name(nodeName), Name("memory"))
```

Comment:
Looks good, though i think a comment for each of these to describe them would be good. What does memory mean here?